.github/workflows/docs.yml

```
@@ -0,0 +1,37 @@
+
+name: Update docs
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

We probably should only run this on our repos and not on forks because they don't have the secrets, same as we did it with the javadoc/wiki push action of ChatOverflow.
```suggestion
    runs-on: ubuntu-latest
    if: startsWith(github.repository, 'codeoverflow-org') # don't run this in forks
```